tests/test_tensor_measurements.py

```diff
@@ -318,6 +318,7 @@ def circuit(a, b, c):
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
 
+@pytest.mark.xfail
```

Comment:
These tests are currently failing and will need to be rewritten. As they are integration tests, I am okay with them being stochastic (i.e., look at the resulting samples, take the mean and variance, and make sure they match <A> and var(A)). However, care should be taken so that the chance they fail stochastically is so small, it will likely never happen.